src/common/ServerCompatBanner.js

```diff
@@ -62,14 +62,14 @@ export default function ServerCompatBanner(props: Props): Node {
       buttons={[
         {
           id: 'dismiss',
-          label: 'Dismiss',
+          label: 'Remind me later',
           onPress: () => {
             dispatch(dismissCompatNotice());
           },
         },
         {
-          id: 'resolve',
-          label: isAdmin ? 'Fix now' : 'Learn more',
```

Comment:
I think these were the only two uses of these two strings, so they can be deleted from `messages_en.json`.